examples/stable-diffusion/image_to_video_generation.py

```diff
@@ -185,44 +219,61 @@ def main():
         "use_hpu_graphs": args.use_hpu_graphs,
         "gaudi_config": args.gaudi_config_name,
     }
+
+    set_seed(args.seed)
     if args.bf16:
         kwargs["torch_dtype"] = torch.bfloat16
 
-    pipeline = GaudiStableVideoDiffusionPipeline.from_pretrained(
-        args.model_name_or_path,
-        **kwargs,
-    )
+    if args.control_image_path is not None:
+        from optimum.habana.diffusers import GaudiStableVideoDiffusionPipelineControlNet
```

Comment:
```suggestion
        from optimum.habana.diffusers import GaudiStableVideoDiffusionControlNetPipeline
```

just a personal suggestion to be compliant with other naming like `GaudiStableDiffusionControlNetPipeline`

Reply:
Done, thanks.